### ISSUE TITLE
Files not being cleaned after test runs

### ISSUE THREAD

**Issue by stevepm** [maximilien/i18n4go#22]
Just wanted to see if this is supposed to happen, or if this was a bug in the test suite.

```
git status
On branch master
Your branch is up-to-date with 'origin/master'.

nothing to commit, working directory clean
  master  ~/go/src/github.com/maximilien/i18n4go 
» ./bin/test

 Cleaning build artifacts...

 Formatting packages...

 Integration Testing packages:
ok  	github.com/maximilien/i18n4go/integration/checkup	1.707s
ok  	github.com/maximilien/i18n4go/integration/create_translations	1.717s
ok  	github.com/maximilien/i18n4go/integration/extract_strings	1.913s
ok  	github.com/maximilien/i18n4go/integration/fixup	1.882s
ok  	github.com/maximilien/i18n4go/integration/merge_strings	1.629s
ok  	github.com/maximilien/i18n4go/integration/rewrite_package	1.989s
ok  	github.com/maximilien/i18n4go/integration/show_missing_strings	1.570s
?   	github.com/maximilien/i18n4go/integration/test_helpers	[no test files]
ok  	github.com/maximilien/i18n4go/integration/verify_strings	1.752s

 Vetting packages for potential issues...

SWEET SUITE SUCCESS
  master ●  ~/go/src/github.com/maximilien/i18n4go 
» git status
On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   test_fixtures/rewrite_package/d_option/expected_output/doption/en.all.json
	modified:   test_fixtures/rewrite_package/d_option/expected_output/doption/test.go.en.json
	modified:   test_fixtures/rewrite_package/d_option/expected_output/doption/test2.go.en.json
```

**Comment by maximilien** [comment]
Cannot repro, so closing

**Comment by maximilien** [comment]
Reopening this issue. Could replicate and have a fix

**Comment by maximilien** [comment]
Fix with commit 35c9159dc0d08464e66fdffe4abeacaaa9b15006
